### DISCUSSION TITLE
Keep sidebar on pages with tabs

### DISCUSSION THREAD

**Discussion by Arctomachine** [payloadcms/payload#8789] [Feature Requests & Ideas]
Currently sidebar breaks when tabs are added to document. All fields from sidebar go into main space.

Use case: seo plugin in tab mode breaks sidebar for main content of document.

Two possible solutions:
- Easy: show sidebar for all tabs
- Harder: show sidebar for fields defined in current tab and higher

**Comment by andershermansen** [comment]
@Arctomachine Maybe you can show your config and screenshots?

This works as expected for us. Our sidebar fields are defined outside of any tab and will always show regardless of which tab you are on.

**Reply by akhrarovsaid** [reply]
Hey @andershermansen,

We had a discussion on Discord about this. I think the intention here is to gauge the community feedback on whether the active tab should show/hide sidebar fields if those aforementioned fields are defined within tabs.

As it stands, if you define a sidebar field deeply nested, such as in a tab, then they will not show up in the sidebar but render in the main area instead.

**Reply by andershermansen** [reply]
@akhrarovsaid Sure, I was just unsure if @Arctomachine understood the current possibilities or not. So I just wanted to inform that at least one way of using tab with sidebar at the same time is already working. If he has problems with this.

**Comment by hades200082** [comment]
Another convo about this on Discord: https://discord.com/channels/967097582721572934/1215659716538273832/1303358797011750942

I see a few different use-cases here:

1. Auto-tabs like with SEO plugin
  In this case I wouldn't expect my sidebar fields to be impacted - just show them in the sidebar as normal. This would require the UI to understand that the tabs were automatically added by the plugin.
2. A tabs field defined as show in sidebar
  Show these tabs and their contents in the sidebar
3. Fields defined as show in sidebar within an explicit tabs field in collection def
  This is the tricky one - do you show tabs in the sidebar that match those in the main view to keep fields separate still, or do you just list all sidebar fields in the sidebar in the order they were defined? tbh, I'm not sure I like either of those options - I'd rather these show a warning in the UI or something instead so I can handle it.

I think the main issue atm is the way it just silently fails and doesn't tell you how to fix it.
